packages/react-core/src/components/DualListSelector/examples/DualListSelector.md

```diff
@@ -103,8 +120,12 @@ class ComplexDualListSelector extends React.Component {
         this.setState(prevState => {
           const available = prevState.availableOptions.sort((a, b) => {
             let returnValue = 0;
-            if (a.props.children > b.props.children) returnValue = 1;
-            if (a.props.children < b.props.children) returnValue = -1;
+            if (a.props.children[0].props.children > b.props.children[0].props.children) {
+              returnValue = 1;
+            }
+            if (a.props.children[0].props.children < b.props.children[0].props.children) {
+              returnValue = -1;
```

Comment:
you need to make this same change to the logic onSort in the chosen panel. :)

Reply:
Good catch, sorry I missed it!